presto-google-sheets/src/test/java/io/prestosql/plugin/google/sheets/TestGoogleSheets.java

```diff
@@ -62,6 +62,8 @@ public void testListTable()
     {
         assertQuery("show tables", "SELECT * FROM (VALUES 'metadata_table', 'number_text', 'table_with_duplicate_and_missing_column_names')");
         assertQueryReturnsEmptyResult("SHOW TABLES IN gsheets.information_schema LIKE 'number_text'");
+        assertQuery("select table_name from gsheets.information_schema.tables WHERE table_schema <> 'information_schema'", "SELECT * FROM (VALUES 'metadata_table', 'number_text', 'table_with_duplicate_and_missing_column_names')");
+        assertQuery("select table_name from gsheets.information_schema.tables WHERE table_schema <> 'information_schema' LIMIT 1000", "SELECT * FROM (VALUES 'metadata_table', 'number_text', 'table_with_duplicate_and_missing_column_names')");
```

Comment:
would we be able to run `io.prestosql.testing.AbstractTestIntegrationSmokeTest#testSelectInformationSchemaTables` and `testSelectInformationSchemaColumns` for Google Sheets?

Reply:
Once we add support for writes in sheets connector we will be able to run these tests. Or if we could push the tpch data into the sheets which is specified in the test then we could run these tests.